### ISSUE TITLE
Could not load CSS

### ISSUE THREAD

**Issue by cleytoncarvalho** [remix-run/remix#10335]
### Reproduction

I have a Remix project that I just updated to vite@6.0.3 and react@19.0.0. I am using Tailwind CSS with PostCSS and loading it directly in my app/root.tsx file:

`import './styles/tailwind.css'`

Before the update, everything worked fine. However, after the update, the page initially loads with no CSS applied, and then after a few milliseconds, it displays the page with the correct CSS. In the terminal, the following message appears:

`Could not load /Users/cleytoncarvalho/Projects/test/app/styles/tailwind.css`

Here’s my package.json file:

"dependencies": {
		"@remix-run/node": "^2.15.1",
		"@remix-run/react": "^2.15.1",
		"@remix-run/serve": "^2.15.1",
		"isbot": "^5.1.17",
		"react": "^19.0.0",
		"react-dom": "^19.0.0",
		"tailwindcss-animate": "^1.0.7"
},
"devDependencies": {
		"@remix-run/dev": "^2.15.1",
		"@remix-run/route-config": "^2.15.1",
		"@remix-run/routes-option-adapter": "^2.15.1",
		"@tailwindcss/typography": "^0.5.15",
		"@types/react": "^19.0.1",
		"@types/react-dom": "^19.0.2",
		"autoprefixer": "^10.4.20",
		"postcss": "^8.4.49",
		"remix-flat-routes": "^0.6.5",
		"tailwindcss": "^3.4.16",
		"typescript": "^5.7.2",
		"vite": "^6.0.3",
		"vite-tsconfig-paths": "^5.1.4"
}

Does anyone know how to fix this issue?

### System Info

```shell
System:
    OS: macOS 15.1.1
    CPU: (8) arm64 Apple M1
    Memory: 66.34 MB / 8.00 GB
    Shell: 5.9 - /bin/zsh
  Binaries:
    Node: 20.15.0 - ~/.asdf/installs/nodejs/20.15.0/bin/node
    npm: 10.7.0 - ~/.asdf/plugins/nodejs/shims/npm
    pnpm: 9.10.0 - ~/Library/pnpm/pnpm
    bun: 1.0.29 - ~/.asdf/shims/bun
    Watchman: 2023.06.12.00 - /opt/homebrew/bin/watchman
  Browsers:
    Chrome: 131.0.6778.140
    Safari: 18.1.1
```

### Used Package Manager

pnpm

### Expected Behavior

CSS should load properly.

### Actual Behavior

CSS does not load properly.

**Comment by robin-siegl** [comment]
i have the same issue, but i'm still on `"react": "^18.2.0"`.  
I'm currently debugging,...

**Comment by simonsmith** [comment]
@robin-siegl I encountered this. Was fixed in `react-router 7.1.0`
